ldml/lt4all.md

```diff
@@ -0,0 +1,71 @@
+---
+title: LT4All 2025 - CLDR Keyboards
+---
+
+## Introducing Unicode CLDR Keyboards
+
+### An open keyboard specification and repository designed for Indigenous languages
+
+## **CLDR Keyboards** are software keyboard layouts
+
+* Desktop and laptop computers
+* Touchscreen devices – phones and tablets
+* IoT devices that support text input
+
+## Benefits for your language community
+
+* CLDR Keyboard WG is creating a clear process for authors to submit keyboard
+  layouts to CLDR
+* Engage with the CLDR team – no need to build multiple relationships with
+  industry vendors
+* Submission process is open and transparent
+* New and updated keyboards will be distributed on a regular schedule to vendors
+* Copyright remains with the creator of the keyboard (usage rights are provided
+  to vendors via CLDR contributors agreement)
+
+## Who is involved?
+
+* [Unicode Consortium][1] – CLDR Keyboard Working Group
+* Major industry vendors are implementing the specification in operating systems
+  and devices
+* [SIL Global][2] develops Keyman – an open source and free implementation for
+  CLDR keyboards
+* [Translation Commons][3] creates keyboards for Indigenous communities
+
+## Designed for Indigenous and low-resource language needs
+
+* Simple, concise high-level keyboard design language
+* Cross-platform by design
+* Minimal reference to hardware
+* Transforms provide complex script support
+  * reordering, recombining, normalization
+  * user-friendly typing experience
+  * produce consistent output
+
+## About CLDR – the Common Locale Data Repository
+
+* A home for all industry vendors to intake locale and cultural data – and now also keyboard layouts
+* Already in use by all major software companies (including Amazon, Apple,
+  Google, Meta, Microsoft etc.) for calendar, language, and regional metadata
+
+## About Keyman
+
+* Free, open source software keyboard – MIT licensed
+* Windows, macOS, Linux, Android, iOS, and web
+
+## How to start creating CLDR Keyboards
+
+* [Read the specification](https://www.unicode.org/reports/tr35/tr35-keyboards.html)
+* [Download Keyman Developer](/developer)
+* [More about Keyman and CLDR/LDML Keyboards](/ldml)
```

Comment:
Maybe can remove this? (will redirect to self, right?)

Reply:
ah, links to the older page.

Reply:
Yes, this is a landing page intended just for the LT4All Conference. We won't promote it outside that venue. The /ldml page remains